tests/components/modbus/conftest.py

```diff
@@ -3,7 +3,7 @@
 import logging
 from unittest import mock
 
-from pymodbus.exceptions import ModbusException
+from pymodbus.exceptions import ModbusException  # type: ignore
```

Comment:
Please remove all the import ignores. They should not be needed.

Reply:
Sorry, I forgot all about tests, after I saw (to my surprise) that mypy only looks in homeassistant not tests.

I have updated test as well.

Regarding you thumbs up on the comment above, please see my response.